docs/install/tutorials/rpi.rst

```diff
@@ -272,22 +272,17 @@ Installing Kolibri
 
 #. Add our Ubuntu PPA with these special instructions:
 
-   .. warning:: The package ``kolibri-server`` is currently undergoing final testing. We are distributing ``kolibri-server`` through a development repository (PPA) called ``kolibri-proposed``, which is a temporary addition to the below commands. This will change in September 2019, removing one of the lines below and the specification of the Kolibri version number in the next step. Please stay tuned for the change.
-
    .. code-block:: bash
 
-      sudo su -c 'echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu xenial main" >> /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list'
-      # The below command will be removed in September 2019, see above warning
-      sudo su -c 'echo "deb http://ppa.launchpad.net/learningequality/kolibri-proposed/ubuntu xenial main" >> /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list'
+      sudo su -c 'echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu xenial main" > /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list'
```

Comment:
I think we should use bionic instead of xenial in the repo names, as cosmic is the latest LTS

Reply:
Done!